client/src/components/MultiJurisdictionAudit/RoundManagement/index.tsx

```diff
@@ -83,14 +83,14 @@ const RoundManagement = ({
   }
 
   const ballotsOrBatches =
-    auditSettings.auditType === 'BATCH_COMPARISON' ? 'batches' : 'ballots'
+    auditSettings.auditType === 'BATCH_COMPARISON' ? 'Batches' : 'Ballots'
```

Comment:
@jonahkagan Shouldn't these be capitalized, since that looks more correct when used on their own, and since you're lowercasing them on line 93?

Reply:
The first letter is capitalized so they can start the sentence on L106. I lowercase them on L93 so they can be used in the middle of a sentence

Reply:
Oh gotcha, I was reading it wrong. Looks good!